src/csharp/Microsoft.Spark/Utils/CommandSerDe.cs

```diff
@@ -181,7 +181,7 @@ private sealed class UdfWrapperData
             List<UdfSerDe.UdfData> udfs)
         {
             UdfSerDe.UdfData udfData = UdfSerDe.Serialize(func);
-            if (udfData.MethodName != UdfWrapperMethodName)
+            if (!Attribute.IsDefined(func.Target.GetType(), typeof(UdfWrapperAttribute)))
```

Comment:
remove `UdfWrapperMethodName` constant as well?

Reply:
oh good catch.

Reply:
Nm.  It's being used in the `CreateUdfWrapperDelegate` method.

Reply:
OK. in this case, we need the `(udfData.MethodName != UdfWrapperMethodName)` check to ensure the name matches up for the wrapper in `CreateUdfWrapperDelegate`.  Can we also rename `ForeachWriterWrapper.Execute` to `ForeachWriterWrapper.Process` or something like that to differentiate? We can leave the attribute check for robustness.

Reply:
Sure, made the changes.